spec/rails_helper.rb

```diff
@@ -0,0 +1,68 @@
+ENV["RAILS_ENV"] ||= 'test'
+
+require 'spec_helper'
+require 'rspec/rails'
+# require 'capybara/rspec'
+require 'database_cleaner'
+require 'draper/test/rspec_integration'
+# require 'email_spec'
+# require 'capybara/poltergeist'
+# require 'capybara/rails'
+# require 'shoulda-matchers'
+
+# Requires supporting ruby files with custom matchers and macros, etc, in
+# spec/support/ and its subdirectories. Files matching `spec/**/*_spec.rb` are
+# run as spec files by default. This means that files in spec/support that end
+# in _spec.rb will both be required and run as specs, causing the specs to be
+# run twice. It is recommended that you do not name files matching this glob to
+# end with _spec.rb. You can configure this pattern with with the --pattern
+# option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
+Dir[Rails.root.join("../../spec/support/**/*.rb")].each { |f| require f }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.